tests/trnsysGUI/storageTank/testConfigureStorageDialog.py

```diff
@@ -1,11 +1,13 @@
 import pathlib as _pl
 
-import PyQt5.QtCore as _core
```

Comment:
All imports in `pytrnsys`/`trnsysGUI` should be of the form `import foo.bar.baz as _alias`. In particular, in new code, we don't import individual classes.